src/main/scala/nl/knaw/dans/easy/archivebag/EasyArchiveBag.scala

```diff
@@ -85,21 +90,64 @@ object EasyArchiveBag {
     }
     val deleted = tempFile.delete()
     if(!deleted) log.warn(s"Delete of $tempFile failed")
-    response.getStatusLine.getStatusCode match {
+    val seIri = response.getStatusLine.getStatusCode match {
       case 201 =>
         val location = response.getFirstHeader("Location").getValue
         log.info("SUCCESS")
         log.info(s"Deposit created at: $location")
         s"${location.split('/').last}/${s.bagDir.getName}"
+
       case _ =>
         val writer = new StringWriter()
         IOUtils.copy(response.getEntity.getContent, writer, "UTF-8")
         log.error(s"Deposit failed: ${response.getStatusLine}, ${writer.toString}")
         throw new RuntimeException("Deposit failed")
     }
+    log.info(seIri)
+    (seIri, getConfirmationState(seIri, http))
+  }
+
+  def getConfirmationState(locationBagDir : String, http : CloseableHttpClient)(implicit s: Settings) : String = {
+    var state = STATE_FINALIZING
+    val protocol = s.storageDepositService.getProtocol
+    val host = s.storageDepositService.getHost
+    val context = s.storageDepositService.getPath.split('/')(1)
+    val location = locationBagDir.split('/')(0)
+    val url = s"${protocol}://${host}/${context}/statement/${location}"
```

Comment:
- It should be possible to create the url with less lines, something like
          
          val url = new URL(s.storageDepositService, s"statement/$location") 
  or something
- What happens if `locationBagDir` is empty?